app_controller/pixi_frontend/src/ts/init_pixi.ts

```diff
@@ -4,14 +4,63 @@ function initalize_application(width: number, height: number){
     
     let Application = PIXI.Application;
     
-    let loader = PIXI.Loader;
-    
     let app = new Application({width: width, height: height});
     
     app.renderer.backgroundColor = 0x895044;
     document.body.appendChild(app.view);
+    app.loader
+        .add([
+            "assets/buttons/red_button_pressed.png",
+            "assets/buttons/red_button_unpressed.png", 
+            "assets/buttons/blue_button_pressed.png",
+            "assets/buttons/blue_button_unpressed.png", 
+            "assets/buttons/yellow_button_pressed.png",
+            "assets/buttons/yellow_button_unpressed.png"
+        ])
+        .load(setup);
+
     return app;
 }
 
-var app = initalize_application(500, 500);
+function setup() {
+
+    const red_button_textures = [app.loader.resources["assets/buttons/red_button_unpressed.png"].texture, app.loader.resources["assets/buttons/red_button_pressed.png"].texture];
+
+    const blue_button_textures = [app.loader.resources["assets/buttons/blue_button_unpressed.png"].texture, app.loader.resources["assets/buttons/blue_button_pressed.png"].texture];
+
+    const yellow_button_textures = [app.loader.resources["assets/buttons/yellow_button_unpressed.png"].texture, app.loader.resources["assets/buttons/yellow_button_pressed.png"].texture];
+
+    init_button(app, red_button_textures, {'x':180, 'y':86}, "UP");
+    init_button(app, blue_button_textures, {'x':50, 'y':250}, "LEFT");
+    init_button(app, blue_button_textures, {'x':310, 'y':250}, "RIGHT");
+    init_button(app, yellow_button_textures, {'x':500 , 'y':150}, 'UPDATE');
+
+}
+
+function init_button(app: PIXI.Application, textures: Array<PIXI.Texture>, pos, value_to_write: string, scale_x:number = .5, scale_y: number = .5, enc: TextEncoder = new TextEncoder()) {
+    
+    let button = new PIXI.Sprite(textures[0]);
+    
+    button.scale.set(scale_x,scale_y);
+    button.position.set(pos.x, pos.y);
+    button.interactive = true;
+    button
+        .on('mousedown', function() {
+            if(characteristic == undefined){
+                console.log("connect to bluetooth first");
```

Comment:
Prefer using the `alert` JS function instead of `console.log` for things that the user should see